### DISCUSSION TITLE
Shouldn't `citar` treat 'First Last' and 'Last, First' the same?

### DISCUSSION THREAD

**Discussion by apc** [emacs-citar/citar#529] [Q&A] [answered]
As things stand, on my end, if I have "Smithers, Waylon" as author for an entry and "Waylon Smithers" as the author for another, the corresponding entries are displayed differently:

<img width="1282" alt="image" src="https://user-images.githubusercontent.com/350504/149228032-31f1a506-5cec-40e0-b4e4-93f213757076.png">

I'd have expected `citar` to list either last names only or first and last, regardless of the order in which they're listed in the `author` field. Is the behavior I'm seeing the intended behavior? 

Thanks again!

**Comment by bdarcus** [comment, marked as answer]
> Is the behavior I'm seeing the intended behavior?

Yes; see the docstring for the function responsible for name shortening.

https://github.com/bdarcus/citar/blob/b24b5b94b7d5f6e7e96331b48a22e977e0013de1/citar.el#L518-L522

Corporate names as in "Some Government Agency".

**Reply by apc** [reply]
Interesting. I guess I've never had need for that, so didn't even think about it. How if at all does e.g. biblatex handle that?

**Reply by bdarcus** [reply]
> How if at all does e.g. biblatex handle that?

Here's what the manual says, which suggests it's consistent with bibtex:

> Corporate authors and editors are given in the author or editor field, respectively.
Note that they must be wrapped in an extra pair of curly braces to prevent data
parsing from treating them as personal names which are to be dissected into their
components.

It could be that the function logic could use some refinement, though we'd need to check how this would impact parsebib's parsing of json.

**Reply by apc** [reply]
Alternatively, a function that you can add as a hook to `bibtex-clean-entry` and which would transform author fields when not in double braces? Not sure what would be best.

**Comment by rudy6912** [comment]
I observe this bug as well. For instance, Knuth shows as "Knuth", "Donald E. Knuth", and "Donald Ervin Knuth" in Citar. In BibTeX, `{baz, foo bar}` magically equals `{foo bar baz}`, but `{{foo bar baz}}` cancels any such magic.

**Reply by bdarcus** [reply]
Not really sure it's a bug. It's just a simple, general (non-bibtex-specific) function ATM.

Feel free to create an issue so we can track it.

**Reply by rudy6912** [reply]
Opened #532. Thanks!
